Backend/dev/integration-tests/run-tests.sh

```
@@ -55,6 +55,8 @@ AIRPORT_DIR="$SCRIPT_DIR/collections/AirportTaxiFlow"
 TOLL_CONFIG_DIR="$SCRIPT_DIR/collections/TollConfigFlow"
 TOLL_RIDE_DIR="$SCRIPT_DIR/collections/TollRideFlow"
 DRIVER_IMAGE_DIR="$SCRIPT_DIR/collections/DriverImageFlow"
+FACEMATCH_DIR="$SCRIPT_DIR/collections/FaceMatchOnboardingFlow"
```

Shellcheck (0.11.0)

[warning] FACEMATCH_DIR appears unused. Verify use (or export if used externally).

(SC2034)